lib/view/sheet/decleration.py

```diff
@@ -380,12 +381,35 @@ def max(initializer=None, converter=None):
         )
 
 
-class BaseProjector(object):
```

Comment:
I refactored and split BaseProjector into Projector and BaseProjector. Now we have Projector -> BaseProjector -> Number,String, .... and Projector -> Any, Sum, Div, ....

Reply:
Having the name "projector" and "BaseProjector" is a little confusing. Maybe "BaseProjector" could be "SimpleProjector" or "ScalarProjector" something that denotes it's use for extracting single values.

Reply:
Yeah, you're right. Ill change the names